README.md

```diff
@@ -34,16 +34,18 @@ to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it sho
 #### Providers
 Currently this document is arranged to support object storage users of the following PaaS solutions:
 
-- Alibaba
-- AWS
-- Azure
-- Google Cloud Platform
-- IBM
-- Oracle
+- Alibaba Cloud (Aliyun): `ALIBABA`
+- Amazon AWS: `AWS`
+- Microsoft Azure: `AZURE`
+- Google Cloud Platform: `GCP`
+- IBM Cloud: `IBM`
+- Oracle Cloud: `ORACLE`
```

Comment:
@m-mohr Should we also document the `OTHER` value here?

Reply:
Yes, that makes sense.